beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -175,10 +175,14 @@ func (s *Service) validateAggregatedAtt(ctx context.Context, signed ethpb.Signed
 		tracing.AnnotateError(span, wrappedErr)
 		return result, err
 	}
+	committee, err := helpers.BeaconCommitteeFromState(ctx, bs, aggregate.GetData().Slot, committeeIndex)
+	if err != nil {
+		return pubsub.ValidationIgnore, err
+	}
 
-	committee, result, err := s.validateBitLength(ctx, bs, aggregate.GetData().Slot, committeeIndex, aggregate.GetAggregationBits())
```

Comment:
why was this removed ? Its effectively the same as what has been added in

Reply:
`validateBitLength` extracted the comittee from state but I need it outside for single attestation. I did this to prevent reading it twice

Reply:
you're right, I will change it back